supporting-blog-content/result-diversification/diversification.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Result diversification with Elasticsearch\n",
+    "This notebook demonstrates:\n",
+    "1. Loading fashion dataset\n",
+    "2. Index in Elasticsearch using image search\n",
+    "3. Search items with a broad search term\n",
+    "4. Apply result diversification with the MMR algorithm to the results.\n",
+    "\n",
+    "Check out our blog post on this topic to learn more about "
```

Comment:
I guess we will update this later with the actual link? once the blog post is available?